lib/multihandle.h

```diff
@@ -139,7 +139,9 @@ struct Curl_multi {
   struct curltime timer_lastcall; /* the fixed time for the timeout for the
                                     previous callback */
   unsigned int max_concurrent_streams;
-
+#if defined(USE_NGHTTP2) || defined(USE_HYPER)
+  unsigned int stream_window_size;
+#endif
```

Comment:
the other uses of stream_window_size are guarded by USE_HTTP2, would that be more appropriate here?

Reply:
Yes!